caddyconfig/httpcaddyfile/builtins_test.go

```diff
@@ -213,3 +214,45 @@ func TestRedirDirectiveSyntax(t *testing.T) {
 		}
 	}
 }
+
+func TestErrorLine(t *testing.T) {
+	for i, tc := range []struct {
+		input     string
+		errorFunc func(err error) bool
+	}{
+		{
+			input: `(t1) {
```

Comment:
Test duplicate import

Reply:
This test is actually just to cover that the error message produced includes the `(import t1)` bit as an indicator to the user of where the problem is.

In both tests here, an error is always produced by the `abort` directive's parsing, because `abort` does not support any arguments at all.

The first import in this test does not produce an error, because the variadic placeholder doesn't produce any tokens, but the second import does produce tokens which causes the `abort` directive to error.